File: AUGUST-2025/05-08-2025/p3477.dart
class Solution {
  int numOfUnplacedFruits(List<int> fruits, List<int> baskets) {
    List<int>bas=[...baskets];
    for(int i=0;i<fruits.length;i++){
      for(int j=0;j<bas.length;j++){
        if(fruits[i]<=bas[j]){
            bas.removeAt(j);
            break;
        }
      }
    }
    return bas.length;
  }
}

File: AUGUST-2025/05-08-2025/p3477.swift
class Solution {
    func numOfUnplacedFruits(_ fruits: [Int], _ baskets: [Int]) -> Int {
        var baskets = baskets
        for i in fruits {
            fetchBasket(i,&baskets)
        }
        return baskets.count
    }
    func fetchBasket(_ fruit:Int, _ basketArr: inout [Int]) {
        for i in 0..<basketArr.count {
            if(basketArr[i] >= fruit) {
                basketArr.remove(at:i)
                break
            }
        }
    }
}